reblog/_toc.yml

```diff
@@ -0,0 +1,7 @@
+- file: intro
+- file: experiments
+  expand_sections: true
+  sections:
+    - file: basics
+    - file: lrmom
+    - file: weight_distribution
```

Comment:
And more... 😃🤪

Reply:
https://jupyterbook.org/customize/toc.html another copy paste